plugins/modules/redhat_subscription.py

```diff
@@ -1074,7 +1076,16 @@ def main():
             'server_port': {},
             'rhsm_baseurl': {},
             'rhsm_repo_ca_cert': {},
-            'auto_attach': {'aliases': ['autosubscribe'], 'type': 'bool'},
+            'auto_attach': {
+                'type': 'bool',
+                'depecated_aliases': [
```

Comment:
The `aliases` needs to stay.

Reply:
ah ok, thanks; it was not clear whether it must be in either of `aliases` and `deprecated_aliases`, or only in one